build.gradle

```diff
@@ -1,3 +1,9 @@
+plugins {
+    id 'signing'
+    id("io.github.gradle-nexus.publish-plugin") version "1.1.0"
```

Comment:
Please use a consistent quotation style in the files. It seems we have more single quotes than double quotes, so let's go with single quotes.